### ISSUE TITLE
Gradients do not exist for variables ['sync_batch_normalization_8/gamma:0', 'sync_batch_normalization_8/beta:0', 'conv3d_21/kernel:0', 'conv3d_21/bias:0']

### ISSUE THREAD

**Issue by fahadahmedkhokhar** [Svdvoort/PrognosAIs_glioma#6]
I am using your custom defined model for training of the data. But before training start following error shows. Can you please address this issue?

 **prognosais WARNING Gradients do not exist for variables ['sync_batch_normalization_8/gamma:0', 'sync_batch_normalization_8/beta:0', 'conv3d_21/kernel:0', 'conv3d_21/bias:0'] when minimizing the loss. If you're using `model.compile()`, did you forget to provide a `loss` argument?**

**Comment by Svdvoort** [comment]
I would need some more information about what you are trying to do: 

- Which custom model are you using?
- What does your training script look like? 

Based on this information my first two suggestions would be:

1. Did you provide a loss in the `model.compile()` as suggested by the error message? 
2. Try lowering your learning learning rate, sometimes this can be the case as well.

**Comment by fahadahmedkhokhar** [comment]
I am using your custom_model.py file. I use the 0.001 learning rate as well but it doesn't work

**Comment by Svdvoort** [comment]
Closed in favor of #7
